spellcheck.py

```diff
@@ -63,30 +83,42 @@ def check():
     if request.method == "POST":
         text = ''
         text = request.form['wrongwords']
-        iwords = text.strip().lower().split()
+        iwords = text.strip().split()
         guesses = []
         r = []
+
+        language = request.form['language']
+
+        if language == 'irish':
+            unique_words = unique_words_irish
+            word_probability = word_probability_irish
+        
+            print('in irish')
+        elif language == 'english':
+            unique_words = unique_words_english
+            word_probability = word_probability_english
+
         for word in iwords:
             guesses = correct_spelling(word, unique_words, word_probability)
-            toporder = sorted(guesses, key=lambda x: x[1], reverse=True)[:len(guesses)]  # arranging suggestions in decreasing order
+            print('after guesses')
```

Comment:
Is this code for your reference to check where it is in your interpreter ?

Reply:
I can delete it if you'd like

Reply:
I deleted it.